### PR TITLE
Add the "User Guide > Create knowledge" documentation pages

### PR DESCRIPTION
**Pull request by Lhorus6**
Add the four documentation pages of the "Created knowledge" section :

- Add Automated import (https://docs.opencti.io/latest/usage/import-automated/)
- Add Manual creations (https://docs.opencti.io/latest/usage/manual-creation/)
- Add Analyst workbench (https://docs.opencti.io/latest/usage/workbench/)
- Complete rewrite of Import from files (https://docs.opencti.io/latest/usage/import-files/)

### CODE REVIEW

File: docs/usage/import-automated.md [hunk 1]
```diff
@@ -1,7 +1,89 @@
...
+- Take deletions into account: Enable this option to delete data from your platform if it was deleted on the providing stream. (Note: Data won't be deleted if another source has imported it previously.)
+- Verify SSL certificate: Check the validity of the certificate of the domain hosting the remote platform.
+- Avoid dependencies resolution: Import only entities without their relationships. For instance, if the stream shares malware, all the malware's relationships will be retrieved by default. This option enables you to choose not to recover them.
+- Use perfect synchronization: This option is specifically for synchronizing two platforms. If an imported entity already exists on the platform, the one from the stream will overwrite it.
+
+![Live stream additional configuration](assets/live-stream-additional-configuration.png)
+
+### TAXII feeds
+
+TAXII Feeds in OpenCTI provide a robust mechanism for ingesting TAXII collections from TAXII servers or other OpenCTI instances. Configuring TAXII ingester involves specifying essential details to seamlessly integrate threat intelligence data. Here's a step-by-step guide to configure TAXII ingesters:
```
Comment (CelineSebe) [inline docs/usage/import-automated.md:50]: TAXII ingestors ?

File: docs/usage/indicators-lifecycle.md [hunk 2]
```diff
@@ -16,23 +17,24 @@ If a data source provides `valid_from` and `valid_until` dates when creating an
...
+| IPv4-Addr and IPv6-Addr               | `TLP:CLEAR` to `TLP:AMBER`       |       30       |
+| IPv4-Addr and IPv6-Addr               | `TLP:AMBER+STRICT` and `TLP:RED` |       60       |
+| IPv4-Addr and IPv6-Addr               | Others                           |       60       |
+| URL                                   | `TLP:CLEAR` to `TLP:GREEN`       |       60       |
+| URL                                   | `TLP:AMBER` to `TLP:RED`         |      180       |
+| URL                                   | Others                           |      180       |
+| Others (e.g. Domain-Name, File, YARA) | All                              |      365       |
 
-### Understanding Time-to-Live (TTL)
+### Understanding time-to-Live (TTL)
```
Comment (CelineSebe) [inline docs/usage/indicators-lifecycle.md:30]: Time-To-Live ?

File: docs/usage/import-automated.md (hunk 1 above)
Comment (Lhorus6) [reply docs/usage/import-automated.md:50]: Perhaps you're right, in which case it needs to be changed in the documentation and in the platform.

File: docs/usage/indicators-lifecycle.md (hunk 2 above)
Comment (Lhorus6) [reply docs/usage/indicators-lifecycle.md:30]: Ok for "Time-To-Live"

**Review by CelineSebe — APPROVED** [review]

**Pull request merged**